tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/types/chat/Extended.kt

```diff
@@ -58,8 +58,8 @@ data class ExtendedChannelChatImpl(
     override val newMembersSeeHistory: Boolean = false,
     @SerialName(maxReactionCountField)
     override val maxReactionsCount: Int = 3,
-    @SerialName(canSendGiftsField)
-    override val canReceiveGifts: Boolean = false,
+    @SerialName(acceptedGiftTypesField)
+    override val acceptedGiftTypes: AcceptedGiftTypes = AcceptedGiftTypes(),
```

Comment:
I have checked [ChatFullInfo](https://core.telegram.org/bots/api#chatfullinfo) and it do not contains info about optionality of this field. Are you sure that it is optional field and must have default value?

Same question about other used default values for `acceptedGiftTypes` fields